src/extension.ts

```diff
@@ -186,17 +186,11 @@ async function setRequestDefaults(): Promise<void> {
 }
 
 export async function deactivateInternal(): Promise<void> {
-    if (!client) {
-        return undefined;
-    }
-    // perform cleanup
-    Configuration.dispose();
-    return await client.stop();
+    // No-op
+    return;
```

Comment:
Can we just get rid of `deactivateInternal()`

Reply:
Yeah, I don't have a strong opinion. We do have to keep `deactivate` inside main.js though.